modules/ROOT/pages/keycloak/binding.adoc

```diff
@@ -5,3 +5,7 @@ include::{partialsdir}/attributes.adoc[]
 = Binding a {mobile-client} with the {service-name} Service
 
 include::{partialsdir}/generic-binding.adoc[]
+
+. You need to choose the *Keycloak client type*. The default is `public` which allows the client to request tokens while type `bearer` should be used to bind to services that only need to verify tokens.
```

Comment:
I think the preferred way is imperative tense `Choose the Keycloak client type` instead of `You need to...`.